### DISCUSSION TITLE
PC goes to sleep while music is playing

### DISCUSSION THREAD

**Discussion by smschumacher** [DeaDBeeF-Player/deadbeef#2975] [General]
Hi,

I have a newly installed NUC13 with Debian Bookworm 12, which I am using as media player. I am using the Gnome Desktop and Deadbeef Version 1.9.5-1. In Energy Profiles I have selected the profile which saves the most power. My problem is that while playing music the desktop goes into suspend mode. I have not found an option or an addon which disables suspend while playing music. 
I know how to disable energy saving but thats not really what I want. What I am looking is the equivalent of the foobar2k component "No Display Standby" https://foobar.hyv.fi/?view=foo_nds.

Yours 
Stefan

**Comment by Oleksiy-Yakovenko** [comment]
I'm fairly sure that it is the job of pulseaudio / audio driver to integrate with system sleep settings, not per-app workarounds. So if this doesn't work, something is wrong in the system configuration.

Perhaps, you should start checking how other audio-playing applications behave, and if they are working properly -- check whether deadbeef is configured to use the correct output plugin (I don't know which one is right for you, depends on the system -- could be ALSA, Pulse or Pipewire).

It can also be that specific energy saving option that you chose overrides sound server behavior, and you need to look in this direction.

**Reply by smschumacher** [reply]
Hi,
as a matter of fact this also happens with Audacious, which I would use in case of Deadbeef not working. I have set different energy saving levels, but this does not make a difference - after some time - longer or shorter depending on the settings - the NUC goes to sleep. 
I have found a workaround but its not very comfortable and sometimes I forget to apply it, which is annoying:
sudo systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
